.github/workflows/cd.yml

```diff
@@ -0,0 +1,45 @@
+name: Rac Lambda CD
+
+on:
+  release:
+    types: [released]
```

Comment:
Doesn't this give us (a blunt, but still) tool for controlling deployment, by making _e.g._ legacy-releases pre-releases only?

Reply:
Yep, that's true